test/shared_test/lib_utility_rate_test.cpp

```diff
@@ -0,0 +1,770 @@
+#include <gtest/gtest.h>
```

Comment:
Did you get a chance to add the "simple" integer (fake rate) test we discussed with @dguittet ?